### PR TITLE
added default language for navigator.language

### PR DESCRIPTION
**Pull request by VivekSingh777**
Submitting a pull request
-------------------------

First of all, thank you for taking the time to contribute to this project :sparkles:!
Every contribution means a lot to us. This is an open source effort, heck yeah.

Before you submit your pull request please read [our contribution guideline](https://github.com/eventzimmer/schema/blob/master/CONTRIBUTING.md).
It includes a `checklist` for the most common questions.

### CODE REVIEW

**Comment by VivekSingh777** [comment]
@Fohlen Done

File: src/store.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ const CATEGORIES_PER_LOCATION = LOCATIONS_ANNOTATED
     accumulator[location.name] = location.tags
     return accumulator
   }, {})
-
+console.log(navigator.language);
```
Comment (Fohlen) [inline src/store.js]: The `console.log` statement shouldn't be here?

File: src/store.js [hunk 2]
```diff
@@ -21,13 +21,13 @@ const CATEGORIES_PER_LOCATION = LOCATIONS_ANNOTATED
...
 const selection = {
   state: {
     categories: [],
     start: null,
     end: null,
-    language: 'de'
+    language: navigator.language.substring(0, 2) || 'de'
```
Comment (Fohlen) [inline src/store.js]: This doesn't really check if the language has previously been set? 

It should be `null` I believe, and in the `main.js` we should check if it is `null`, and if it is, set the `navigator.language` setting!

**Review by Fohlen — CHANGES_REQUESTED** [review]

File: src/store.js (hunk 2 above)
Comment (VivekSingh777) [reply src/store.js]: let me check on that

Comment (Fohlen) [reply src/store.js]: If you need more help on the matter, let me know, I'd be happy to finish this PR 💨

Comment (VivekSingh777) [reply src/store.js]: @Fohlen Looking into the same

**Comment by VivekSingh777** [comment]
@Fohlen Update the code now. Please review

**Comment by Fohlen** [comment]
This looks good, I'll be merging asap! You may add yourself to the [list of contributors](https://github.com/eventzimmer/schema/blob/master/CONTRIBUTORS.md) as you like!

**Pull request merged**